recipes/pyeo/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "pyeo" %}
+{% set version = "0.65" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/clcr/pyeo/archive/v0.65-alpha.tar.gz
```

This URL should contain the version to allow the update bot to work:
```suggestion
  url: https://github.com/clcr/pyeo/archive/v{{ version }}.tar.gz
```